scripts/docker/install_additional_dependencies.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Where is this file used?

Reply:
Good point. I double-checked and I had a typo in PROD Dockerfile:

```
    if [[ -n "${ADDITIONAL_PYTHON_DEPS}" ]]; then \
        /scripts/docker/install_addtional_dependencies.sh; \
    fi;
```

So it was not used. Fixed the typo :)